samples/notebooks/prediction/SDK_Custom_Predict_SDK_Integration.ipynb

```diff
@@ -901,12 +832,238 @@
+    "1. A user account, which is associated with a Google account and is intended to represent a specific individual on your project.\n",
+    "\n",
+    "1. A service account, which is intended to represent an application or a process instead of a person. Use service accounts in your automated build, test, and deployment processes, especially if multiple developers can run these processes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Option 1. Use Google user credentials**\n",
+    "\n",
+    "First authorize gcloud to access the Cloud Platform with Google user credentials. This step requires to put user inputs. If you are using a non-interactive shells, you should open a terminal to run this command. If you are using Vertex Workbench, you could open a terminal to run this command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud auth login"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Grant roles to the user account. Since we will be using this user account to download the model artifacts from Google Cloud Storage, we grant the user account **Storage Object Admin** role. See [IAM roles for Cloud Storage](https://cloud.google.com/storage/docs/access-control/iam-roles) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "USER_ACCOUNT = \"[USER_ACCOUNT]\"  # @param {type:\"string\"}\n",
+    "\n",
+    "!gcloud projects add-iam-policy-binding $PROJECT_ID \\\n",
+    "    --member=user:$USER_ACCOUNT --role=roles/storage.objectAdmin"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Look for the credential file created. List the gcloud info and look for `User Config Directory`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud info | grep \"User Config Directory\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "List all the files under `User Config Directory` to find the credentials."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ls -R [USER_CONFIG_DIRECTORY]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Specify the credential file, which is a json file, as the path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CREDENTIALS_FILE = \"[CREDENTIALS_FILE]\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you are using the user credentails, specify project id so we can pass the project id to the container while we run it."
```

Comment:
typo - credentails

Since this is under Option 1. Use Google user credentials, I think we can just say "Initialize the Vertex AI SDK with the project ID, which is required for using user credentials."

Reply:
Done.